apps/examples/src/examples/Filters/FeGaussianBlur.tsx

```diff
@@ -2,12 +2,12 @@ import React, {Component} from 'react';
 import {Circle, FeGaussianBlur, Filter, G, Svg} from 'react-native-svg';
 
 class StdDeviation5Example extends Component {
-  static title = 'stdDeviation="5"';
+  static title = 'stdDeviation="3"';
```

Comment:
Example titles don't seem to be correct right now. Also does `Android` work ootb?

Reply:
Fixed in [ff942ab](https://github.com/software-mansion/react-native-svg/pull/2388/commits/ff942abcb8b4d68baf60d16376d617179674ea5b)

Reply:
Android blur is limited by the platform and is match weaker anyway. With the full support PR it should be the same as on other platforms.